pubspec.yaml

```diff
@@ -15,15 +15,15 @@ dependencies:
   equatable: ^2.0.5
   flutter:
     sdk: flutter
-  flutter_webrtc: 0.10.6
+  flutter_webrtc: ">=0.10.6"
```

Comment:
I think we did that because of next version of `flutter_webrtc` has problem, maybe we should explicit the version.